### PR TITLE
Added API functionality

### PR DESCRIPTION
**Pull request by singhcoder694**
Added API of getResources from cloudinary in backend. #1 

## Summary by CodeRabbit

- **New Features**
  - Implemented a new route `/getresources` to facilitate the retrieval of resources data.

- **Refactor**
  - Streamlined resource data retrieval by moving logic to a dedicated controller file.

- **Documentation**
  - Updated routing documentation to reflect new endpoint usage.

### CODE REVIEW

File: controllers/getResourcesDBApis.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "Competitive-Programming",
+    "Web-Development",
+    "Programming-Languages",
+    "AI-and-ML",
+    "Database",
+    "Security",
+    "Version-Control",
+    "Mobile-Development",
+    "Cloud-DevOps",
+]
```
[the remark this answers is not in the document]
Comment (singhcoder694) [reply controllers/getResourcesDBApis.js:14]: It is important as it is base for implementation of API

File: controllers/getResourcesDBApis.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            });
+            result.push(resourcesData);
+            return resourcesData;
+            })
+        );
+        return resourcesData;
+    }
+    catch(error){
+        console.log(error);
+    }
```
[the remark this answers is not in the document]
Comment (singhcoder694) [reply controllers/getResourcesDBApis.js:34]: Will be handled in future.

File: routes/addResource_routes.js [hunk 3]
```diff
@@ -19,4 +20,18 @@ router.post(
...
+  "/getresources",
+  async (req, res) => {
+    try {
+      const data=await getResourcesDBApis.getResources(req, res);
+      res.json(data);
+    } catch (error) {
+      res.status(500).json({
+        success: false,
+        message: "It's not you. It's on us. We're working on it",
+      });
```
[the remark this answers is not in the document]
Comment (singhcoder694) [reply routes/addResource_routes.js:33]: It is like this only to say user it's not their falut.

File: sample.env [hunk 4]
```diff
@@ -1,18 +0,0 @@
-DATABASE_URL
```
Comment (ank1traj) [inline sample.env:1]: Why deleted this file?

File: controllers/getResourcesDBApis.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    "Web-Development",
+    "Programming-Languages",
+    "AI-and-ML",
+    "Database",
+    "Security",
+    "Version-Control",
+    "Mobile-Development",
+    "Cloud-DevOps",
+]
+let result=[];
```
[the remark this answers is not in the document]
Comment (ank1traj) [reply controllers/getResourcesDBApis.js:15]: Please go ahead with this.

**Review by ank1traj — CHANGES_REQUESTED** [review]

**Pull request merged**